dev-notes.md

```diff
@@ -0,0 +1,32 @@
+
+
+## Development Notes
+We are iteratively developing a proof of concept in the single-server setting. In the first phase, we focus on basic support for [arbitrary secrets](#arbitrary-secrets-poc). In the second phase, we extend the system to support [signing keys](#signing-keys).
```

Comment:
Minor comment... I think the `arbitrary-secrets-poc` relative link needs to be updated to `arbitrary-secrets` to match the H3 reference.